Documentation/async-disaster-recovery.md

```diff
@@ -0,0 +1,460 @@
+# Failover and Failback in RBD Async Disaster Recovery
+
+[RBD mirroring](https://docs.ceph.com/en/latest/rbd/rbd-mirroring/)
+ is an asynchronous replication of RBD images between multiple Ceph clusters.
+ This capability is available in two modes:
+
+* Journal-based: Every write to the RBD image is first recorded
+ to the associated journal before modifying the actual image.
+ The remote cluster will read from this associated journal and
+ replay the updates to its local image.
+* Snapshot-based: This mode uses periodically scheduled or
+ manually created RBD image mirror-snapshots to replicate
+ crash-consistent RBD images between clusters.
+
+## Create RBD Pools
+
+In this section we create specific rbd pools that are RBD mirroring
+ enabled for use with the DR use case.
+
+> :memo: **Note:** It is also feasible to edit existing pools and
+> enable them for replication.
+
+Execute the following steps on each peer cluster to create mirror
+ enabled pools:
+
+* Create a RBD pool that is mirroring enabled by adding the section
+ spec.mirroring in the CephBlockPool CR, example:
+
+    ```yaml
+    apiVersion: ceph.rook.io/v1
+    kind: CephBlockPool
+    metadata:
+      name: replicapool
```

Comment:
How about we create a pool-mirrored.yaml in the examples folder and name it like this? We try to follow this pattern of yaml files in the examples folder for all doc examples.
```suggestion
      name: mirroredpool
```

Reply:
Sure, will add that.